src/striveneditor.css

```diff
@@ -8,6 +8,9 @@
 */
 
 /** Editor **/
+.StrivenEditor {
+     border: 1px solid #ddd;
+}
 
 .Striven-Editor.editor {
```

Comment:
@wokrabi It looks like this is the class we want to change. Please make the change here and try testing it.